index.js

```diff
@@ -629,6 +629,18 @@ const publicApi = {
         return this;
     },
 
+    /**
+     * If enabled, the Aurelia plugin is enabled
+     *
+     * @param {function} aureliaLoaderOptionsCallback
+     * @returns {exports}
+     */
+    enableAurelia(aureliaLoaderOptionsCallback = () => {}) {
```

Comment:
If it's a plugin, we try to include that in the name to help be transparent. So, `enableAureliaPlugin`